pkgs/by-name/lo/logiops_0_2_3/package.nix

```diff
@@ -11,7 +11,8 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-1v728hbIM2ODtB+r6SYzItczRJCsbuTvhYD2OUM1+/E=";
   };
 
-  PKG_CONFIG_SYSTEMD_SYSTEMDSYSTEMUNITDIR = "${placeholder "out"}/lib/systemd/system";
+  PKG_CONFIG_SYSTEMD_SYSTEMDSYSTEMUNITDIR =
```

Comment:
Since we are not changing anything else in this file, we should keep the formatting

Reply:
If the autoformatter insists.. No point fighting it

Reply:
We do not have an auto formatter and we also do not enforce outdated 80s line lengths.